Docs/Getting-Started-STT_and_TTS.md

```
@@ -155,14 +155,44 @@ Troubleshooting
 - Missing dependencies
   - kokoro_onnx: `pip install kokoro-onnx`
   - onnxruntime: `pip install onnxruntime` (or `onnxruntime-gpu`)
-  - phonemizer / espeak-phonemizer: `pip install phonemizer espeak-phonemizer`
+ - phonemizer / espeak-phonemizer: `pip install phonemizer espeak-phonemizer`
```

markdownlint-cli2 (0.18.1)

Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)